src/Raven.Client/Documents/Smuggler/DatabaseSmugglerImportOptions.cs

```diff
@@ -17,10 +20,12 @@ public DatabaseSmugglerImportOptions(DatabaseSmugglerOptions options)
         }
 
         public bool SkipRevisionCreation { get; set; }
+        public List<string> Collections { get; set; }
     }
 
     internal interface IDatabaseSmugglerImportOptions : IDatabaseSmugglerOptions
     {
         bool SkipRevisionCreation { get; set; }
+        public List<string> Collections { get; set; }
```

Comment:
should be moved to IDatabaseSmugglerOptions from IDatabaseSmugglerExportOptions I think?

Reply:
Done